src/collective/embeddedpage/browser/embeddedpage.py

```diff
@@ -1,9 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
@wolbernd more nitpicking and not a blocker for the merge. The coding line is not necessary any longer since the code base is Python 3 only.